src/main/webapp/babel-src/util.js

```diff
@@ -8,7 +8,7 @@
 export const SEASON_NAMES_PRETTY = ["Fall", "Winter", "Summer"];
 export const SEASON_NAMES = SEASON_NAMES_PRETTY.map((season) => season.toLowerCase());
 export const DEFAULT_PROGRAM = "SOEN-General-Coop";
-export const EXPORT_TYPES = ["PDF", "CSV"];
+export const EXPORT_TYPES = ["PDF", "MD", "TXT"];
 
 // All hardcoded pieces of text which are directly displayed to the user
 export const UI_STRINGS = {
```

Comment:
let's rename `PRORGAM_LIST_LOADING` to just `LOADING_MSG`, since it holds "Loading..." which is general enough to apply  yo anything, no?